### PR TITLE
Post_added

### PR DESCRIPTION
**Pull request by AdarshSingh223**
### All Submissions:

* [ ] Have you followed the guidelines in our Contributing document?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [ ] Does your submission pass tests?
2. [ ] Have you lint your code locally prior to submission?

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: _pages/author-adarsh.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Adarsh Singh"
+layout: default
+permalink: "/author-adarsh-singh"
```
Comment (AvikantSrivastava) [inline _pages/author-adarsh.html:4]: `permalink: "/author-adarsh-singh"` should match with the filename

## Changed that you can make 
1. Either make it `permalink: "/author-adarsh"` 
2. or change the filename to  `_pages/author-adarsh-singh.html`

**Review by AvikantSrivastava — CHANGES_REQUESTED** [review]
Rest of the things look perfect, just make the changes that I have requested :)

**Comment by AvikantSrivastava** [comment]
@AdarshSingh223 Also, while making a PR, don't work on the `master` branch of your cloned repo,
Always make a new branch (copy of the master branch) and makes changes to the new branch.

Later make a PR from the new branch.

This should be avoided
image

**Review by KaushikiKrity — CHANGES_REQUESTED** [review]
The site link you should change! It belongs to someone else.

**Review by KaushikiKrity — CHANGES_REQUESTED** [review]
Your permalink is not working.

**Review by KaushikiKrity — COMMENTED** [review]
This is perfect!

**Comment by AdarshSingh223** [comment]
@kaushikikrityy @AvikantSrivastava I made the recommended changes.

**Comment by KaushikiKrity** [comment]
@AdarshSingh223 The changes look perfect! @AvikantSrivastava please review!

File: Gemfile.lock [hunk 2]
```diff
@@ -1,68 +1,73 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    addressable (2.7.0)
```
Comment (AvikantSrivastava) [inline Gemfile.lock:4]: Please remove this file from the commit

File: _config.yml [hunk 3]
```diff
@@ -135,6 +135,14 @@ authors:
     email:          adarshkgupta@outlook.com
     twitter:        https://www.instagram.com/adkgup/
 
+  adarsh:
```
Comment (AvikantSrivastava) [inline _config.yml:138]: Can you put you name in Alphabetical order?

**Review by AvikantSrivastava — CHANGES_REQUESTED** [review]

**Pull request closed without merging**